config/initializers/devise.rb

```diff
@@ -266,7 +266,7 @@
   # config.navigational_formats = ['*/*', :html, :turbo_stream]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
Any reason for this change? Asking because [in the sign out link a bit above](https://github.com/ARTurleite6/bookshelf_ror/pull/1/files#diff-ecb32a9b0d6ec13e981a5b858b462c5c20f35f5d66df3a41955f82dd731e1148R12) you're using `method: :delete`, so I would expect devise to require you to have ´:delete´ here.